examples/generic_connector_configs/README.md

```diff
@@ -278,20 +279,75 @@ The configuration file for the OAuth 2.0 API can be found at
 
 #### How to use this connector
 * Edit the supplied service configuration to get your OAuth token
-* Run secretless with the supplied configuration(s)
+* Run Secretless with the supplied configuration(s)
 * Query the API using `http_proxy=localhost:8071 curl <Your OAuth2 API Endpoint URL>/{Request}`
 
 ___
 
+### SendGrid Web API
+This example can be used to interact with the
+[SendGrid Web API](https://sendgrid.com/docs/api-reference/).
```

Comment:
I **think** this URL and the one down on line 294 should explicitly point to the reference for API Version 3:
https://sendgrid.com/docs/API_Reference/api_v3.html
I'm expecting that the generic API reference URL will point to whatever latest API version that SendGrid supports.

Reply:
Just found that! I tried googling it earlier, but it brought me to that link there. 

I fixed it :).

Reply:
Looks like the URL was fixed for line 295, but not on line 289. We need the `api_v3.html` path for both, right?